src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+function parseSalary(salary) {
+  return parseInt(salary.replace('$', '').replace(',', ''));
```

Comment:
Consider adding error handling in the `parseSalary` function to ensure the input string is in the expected format. This will prevent potential errors if the salary string does not contain a dollar sign or a comma.